src/components/ProgramViewer.tsx

```
@@ -0,0 +1,79 @@
+interface ProgramViewerProps {
+  sections: Section[];
+}
+
+export const ProgramViewer: React.FC<ProgramViewerProps> = ({ sections }) => {
+  // Keep track of bookmarked section IDs
+  const [bookmarked, setBookmarked] = useState<Record<string, boolean>>({});
+  
+  const defaultTab = sections.length > 0 ? sections[0].id : '';
+
+  const toggleBookmark = (id: string) => {
+    setBookmarked(prev => ({ ...prev, [id]: !prev[id] }));
```

ESLint

[error] Replace `(prev` with `((prev)`

(prettier/prettier)